catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/BrickTestUtils.java

```diff
@@ -68,19 +68,20 @@ public static void checkIfBrickAtPositionShowsString(int position, String string
 				.check(matches(isDisplayed()));
 	}
 
-	public static void checkIfSpinnerOnBrickAtPositionShowsString(int spinnerResourceId, int position, int
-			stringResourceId) {
+	public static void clickAndSelectFromSpinnerOnBrickAndCheckIfSpinnerShowsString(int spinnerResourceId, int
```

Comment:
please rename to "clickSelectCheckSpinnerValueOnBrick" which is still reasonably clear and a lot shorter